js/core/devices.d.ts

```diff
@@ -60,6 +60,8 @@ export interface Device {
     version?: Array<number>;
 }
 
+type EventNames = 'orientationChanged';
```

Comment:
Consider using singular
```suggestion
type EventName = 'orientationChanged';
```